app/views/welcome/index.html.slim

```diff
@@ -0,0 +1,89 @@
+nav.navbar
+  .container-fluid
+    .navbar-header
+      a.navbar-toggle.collapsed aria-expanded="false" data-target="#navbar-collapse" data-toggle="collapse" href="javascript:void(0);" 
+      a.bars href="javascript:void(0);" 
+      a.navbar-brand href="http://lmgtfy.com/?q=nicher%2Br"  NICHER+R
+      = ENV['RAILS_ENV']
+    #navbar-collapse.collapse.navbar-collapse
+        ul.nav.navbar-nav.navbar-right
+            li
+              a[href="/items#index"]
+                i.material-icons
+                  | apps
+                small My Items
+            li
+              a[href="/users/sign_in"]
+                i.material-icons
+                  | exit_to_app
+                small Sign in
+
+body
```

Comment:
Could you explain what we want to add with this body?

Reply:
In this body we will have widgets with statistics from nicher such as amount of items, niches, new users, etc. Also it will contain table with free items.